app/controllers/nodes_controller.rb

```diff
@@ -1,18 +1,17 @@
 # This controller exposes the REST operations required to manage the Node
 # resource.
 class NodesController < NestedNodeResourceController
+  include DynamicFieldNamesCacher
   include NodesSidebar
 
   skip_before_action :find_or_initialize_node, only: [ :sort, :create_multiple ]
   before_action :initialize_nodes_sidebar, except: [ :sort, :create_multiple ]
+  before_action :set_evidence_default_columns, only: :show
+  before_action :set_columns, only: :index
```

Comment:
I think this is a typo? We don't even have an #index action

Reply:
I think it's some bad copy/paste and weird expectations. The method should be `set_columns, only: :show` but it was "working" because the method is named wrong, and already has a before action. I'll make this consistent with `set_columns`